src/utils/responseGetter.js

```diff
@@ -1,8 +1,26 @@
 module.exports = {
 	getMessageResponse,
+	getConfirmationMessage,
 }
 
-async function getMessageResponse(message, messageFilter, timeout = 30000) {
-	message = await message
+async function getMessageResponse(content, msg, timeout = 30000) {
+	const messageFilter = response => response.author.id == msg.author.id
+	
+	const message = await msg.channel.send(content)
 	return (await message.channel.awaitMessages(messageFilter, { max: 1, time: timeout })).first()
+}
+async function getConfirmationMessage(message, content) {
```

Comment:
Due to this having a similar use to `getMessageResponse` it would be best to keep the arguments between them consitent. This would mean adding the optional argument `timeout` and having the same order for `message` and `content` (either way is fine, but I would have message first as it acts on the message)

Reply:
Oops, didn't notice this. Fixed in c85d1775aedaa4ad904bdd6c7e7efbf737a944a6